src/olgm/herald/TileSource.js

```diff
@@ -224,7 +224,7 @@ class TileSourceHerald extends SourceHerald {
       }
     }
 
-    let result = getTileUrlFunction(ol3Coords, 1, proj);
```

Comment:
I suppose the variable at 159 is no longer useful.  Remove?

Reply:
updated as requested

Reply:
updated as requested